### PR TITLE
feat: upgrade to Olive

### PR DESCRIPTION
**Pull request by regisb**

### CODE REVIEW

**Comment by arbrandes** [comment]
Already in there!

**Comment by arbrandes** [comment]
Oh, wait, is this where we track when the olive branch goes into master?  Whoops, reopening.

**Comment by arbrandes** [comment]
@regisb, how does one get this onto the demo instance?  The olive branch is up to date with nightly as I write this.

**Comment by regisb** [comment]
I started a build + launch, it should be ready in a few minutes.

File: tutormfe/patches/local-docker-compose-prod-services [hunk 1]
```diff
@@ -4,3 +4,5 @@ mfe:
     build:
```
Comment (regisb) [inline tutormfe/patches/local-docker-compose-prod-services:4]: Could you remove this build context? It is still causing a rebuild on every `tutor local start`, though we don't need it anymore.

**Comment by ghassanmas** [comment]
course authroing needs this openedx/frontend-app-course-authoring/pull/389 I think we forgat to back port it.

**Pull request merged**